frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -9,6 +9,7 @@ import { toast } from 'react-toastify';
 import { PointFeature } from '../types';
 import PointClusterer from './PointClusterer';
 import { useApi } from 'hooks/useApi';
+import _ from 'lodash';
```

Comment:
You can do `import debounce from 'lodash/debounce';` since we only care about debounce